monitoring/bin/deploy_monitoring_tenant_openshift.sh

```diff
@@ -258,7 +258,7 @@ if [ ! "$OPENSHIFT_AUTH_ENABLE" == "true" ]; then
   fi
 fi
 
-deployV4MInfo "$VIYA_NS" "v4m-tenant-$VIYA_TENANT"
```

Comment:
See note below raising question about naming things for the namespace and tenant levels.